src/java/org/apache/cassandra/io/util/SequentialWriter.java

```diff
@@ -236,14 +221,11 @@ protected void doFlush(int count)
             bytesSinceTrickleFsync += buffer.position();
             if (bytesSinceTrickleFsync >= option.trickleFsyncByteInterval())
             {
-                sync = true;
+            syncDataOnlyInternal();
```

Comment:
Nit: Indent is wrong. Something must be wrong with your IDE setup.

Reply:
no, it is because I've been applying some changes using comparison with clipboard with ignored formatting